python/ray/serve/_private/router.py

```diff
@@ -894,14 +895,18 @@ def __init__(
         deployment_name: str,
         event_loop: asyncio.BaseEventLoop = None,
         _use_new_routing: bool = False,
+        _router_cls: str = "",
     ):
         """Used to assign requests to downstream replicas for a deployment.
 
         The scheduling behavior is delegated to a ReplicaScheduler; this is a thin
         wrapper that adds metrics and logging.
         """
         self._event_loop = event_loop
-        if _use_new_routing:
+
+        if _router_cls:
+            self._replica_scheduler = _load_class(_router_cls)()
```

Comment:
can you add a log statement indicating the class name used similar to below. or combine them all into one statement like:
```python
logger.info(f"Using {self._replica_scheduler.__class__}.", extra={"log_to_stderr": False})
```